fixtures.ttar

```diff
@@ -1,4 +1,4 @@
-# Archive created by ttar -c -f fixtures.ttar fixtures/
+# Archive created by ttar -cvf fixtures.ttar fixtures
```

Comment:
If you use `make update_fixtures` to generate this, then you should get the original line.  I just don't want this one to change if it's going to change back on the next PR.

Reply:
yes, you make point. I will regenterate for next push.